### PR TITLE
Add reviewers POST API for adding reviewrs to a PR (WIP)

### PR DESCRIPTION
**Pull request by bobrowadam**

### CODE REVIEW

**Comment by vermiculus** [comment]
Why was this closed?

**Comment by bobrowadam** [comment]
I just noticed that its already implemented...

**Comment by vermiculus** [comment]
`/repos/:owner/:repo/collaborators` wasn't, but I can add that back in. Thanks anyway!

**Comment by bobrowadam** [comment]
Oh right!
Ok so I'll open it again after I'll test the magithub change I did. Will proceed tomorrow (its 4 am over here...)

File: ghub+.el [hunk 1]
```diff
@@ -420,6 +420,7 @@ logins) from the issue."
     `((assignees . ,(ghubp-get-in-all '(login) users)))))
 
 
+
```
Comment (vermiculus) [inline ghub+.el]: Looks like a spurious change

File: ghub+.el [hunk 2]
```diff
@@ -633,6 +634,12 @@ By default, Issue Comments are ordered by ascending ID."
 
 ;;; Pull Request Reviews:
 
+(defapiget-ghubp "/repos/:owner/:repo/collaborators"
+  "List colaborators.
```
Comment (vermiculus) [inline ghub+.el]: `colaborators` => `collaborators` :-) (ditto for below)

**Review by vermiculus — CHANGES_REQUESTED** [review]
Changes look good! Just a few notes.

Can you squash and rebase onto the current `master`?

**Comment by bobrowadam** [comment]
Squashed and rebased on `master`.

**Review by vermiculus — APPROVED** [review]

**Review by vermiculus — APPROVED** [review]

**Comment by vermiculus** [comment]
I don't know what's going on with review approval (I can't even approve manually via the API), so… 'approved'. :-)

**Pull request merged**